Scripts/Python/stupStartUp.py

```diff
@@ -104,3 +108,14 @@ def OnServerInitComplete(self):
             PtShowDialog("GUIDialog04b")
         else:
             PtShowDialog("GUIDialog06")
+
+        self.randomizeBackground()
+
+    
+    ###########################
+    def randomizeBackground(self):
+        if ilmList := BGObj.sceneobject.getImageLibMods():
+            ilm = ptImageLibMod(ilmList[0])
+            bgChoice = random.choice(ilm.getNames())
+            if newImage := ilm.getImage(bgChoice):
```

Comment:
The API usage here is a bit clunky, IMO. `getImageLibsMods` isn't actually returning an `ImageLibMod`, but rather its key. We also seem to be spin-washing through the name of images when pyILM could return the list of keys of images (or the images themselves) and we could select on those instead?

Reply:
I'm open to any suggestions for a better API to expose for this.  It was expedient to write it this way (and matched existing code), but I'd rather write it to be designed well in the long term if possible.  I based this on the existing pattern provided by `getResponders` and `getPythonMods` for consistency, and they both return lists of keys as well.

Would you prefer I streamline this one?  It would definitely make for better scripting.

Reply:
I would definitely prefer some streamlining. One of the things I desperately want to do is fix some of the more frustrating things about the Python API, like where you ask for a GUI control, and it gives you the key of the control, which you then have to use to the correct object. I think it would make more sense for `getImageLibMods` to return the final image library Python object directly. 😄

Reply:
Alright, I'll see what I can do.

Reply:
Updated!